dev/Scroller/Scroller.cpp

```diff
@@ -6726,19 +6750,22 @@ bool Scroller::RaiseBringingIntoView(
     double targetZoomedHorizontalOffset,
     double targetZoomedVerticalOffset,
     const winrt::BringIntoViewRequestedEventArgs& requestEventArgs,
-    int32_t offsetsChangeId)
+    int32_t offsetsChangeId,
+    _Inout_ winrt::SnapPointsMode* snapPointsMode)
```

Comment:
Would it be appropriate to use a reference here instead of a pointer?

Reply:
I think using a pointer makes it clearer that it's an out parameter. I thought that was the recommended convention too.